### PR TITLE
User reply comment

### PR DESCRIPTION
**Pull request by NguyenThiHangB**

### CODE REVIEW

**Comment by NguyenThiHangB** [comment]
ready
Link task: https://dev.framgia.com/issues/184242

**Comment by NguyenThiHangB** [comment]
ready

**Comment by mdmahmudurrahman** [comment]
commented
Please reset database, try the functionality like as category destroy, making comment, 
fix the problem.

**Comment by NguyenThiHangB** [comment]
ready

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -42,10 +42,17 @@ def destroy
     end
   end
 
+  def new_reply
+    respond_to do |format|
+      format.html {redirect_to @comment.book}
```
Comment (huuphuc93) [inline app/controllers/comments_controller.rb]: format.html {redirect_to @comment.book} em có sử dụng k nhỉ?

**Comment by huuphuc93** [comment]
commented
comment nhỏ

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (NguyenThiHangB) [reply app/controllers/comments_controller.rb]: em có sử dụng ạ

**Comment by NguyenThiHangB** [comment]
ready

**Pull request merged**